app/views/passwords/forgot_password.html.haml

```diff
@@ -0,0 +1,14 @@
+%section.forgot_password.container
+  .row
+    .col-sm-10.col-bg-offset-1
+      = form_tag request_token_path, class: "sign_in" do
```

Comment:
@brandoncc this class comes directly from the ui, I did not add it on my own :) but I agree with you